README.rst

```diff
@@ -1,6 +1,22 @@
 ITKVariationalRegistration
 ==========================
 
+.. image:: https://circleci.com/gh/InsightSoftwareConsortium/ITKVariationalRegistration.svg?style=shield
+    :target: https://circleci.com/gh/InsightSoftwareConsortium/ITKVariationalRegistration
+
+.. image:: https://travis-ci.org/InsightSoftwareConsortium/ITKVariationalRegistration.svg?branch=master
+    :target: https://travis-ci.org/InsightSoftwareConsortium/ITKVariationalRegistration
+
+.. image:: https://img.shields.io/appveyor/ci/InsightSoftwareConsortium/itkvariationalregistration.svg
+    :target: https://ci.appveyor.com/project/InsightSoftwareConsortium/itkvariationalregistration
```

Comment:
This should point to

https://ci.appveyor.com/project/InsightSoftwareConsortium/itkvariationalregistration

Reply:
I do not see the typo :grimacing:

Reply:
Awesome. Thanks. Done in 8af03e9.